Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,58 @@
+require "find"
+
+module Utils
+  class DiskUsage
+    attr_accessor :size_in_bytes, :file_count, :path, :exclude
+
+    def initialize(path, exclude=[])
+      @size_in_bytes = 0
+      @file_count = 0
+      @path = path
+      @exclude = [".DS_Store"] + exclude
+      begin
+        Find.find(@path).each do |f|
+          if !File.directory?(f) and !f.match(/#{@exclude.join('|')}/)
+            @file_count += 1
+            @size_in_bytes += File.size(f)
+          end
+        end
+      rescue Errno::ENOENT
+        raise "DiskUsage: No such path `#{@path}'"
+      rescue
+        raise "DiskUsage: error in calculating disk space!"
+      end
+    end
+
+    def inspect
+      "#{@path} (#{@file_count} files, #{DiskUsage.disk_space_readable(@size_in_bytes)})"
+    end
+
+    def self.disk_space_readable(number)
+      len = number.to_s.length
+      case
+      when len > 9
+        sym, pow = ['G', 3]
```

Comment:
Super nit, but please stick to double quotes here. We've pretty much eradicated single quotes in the core code.